### PR TITLE
added svelte kit starter sample with builder

### PR DESCRIPTION
**Pull request by gustavocodess**
## Description

Add a short description of what changes you made, why you made them, and any other context that you think might be helpful for someone to better understand what is contained in this pull request. This sort of information is useful for people reviewing the code, as well as anyone from the future trying to understand why changes were made or why a bug started happening.

_Screenshot_
If relevant, add a screenshot or two of the changes you made.

### CODE REVIEW

File: examples/svelte-starter-kit/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# create-svelte
```
Comment (samijaber) [inline examples/svelte-starter-kit/README.md:1]: I think it's important & useful to cleanup the README a bit so it also includes info about Builder. See https://github.com/BuilderIO/builder/tree/main/examples/svelte/sveltekit , you can probably copy/paste most of it for this one

File: examples/svelte-starter-kit/src/lib/Builder.svelte [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	import * as BuilderSDK from '@builder.io/sdk-svelte';
+
+	const CUSTOM_COMPONENTS = [
+		...CustomComponents,
+	];
+	// TODO: enter your public API key
+	const BUILDER_PUBLIC_API_KEY = 'bcda49ef60db482bbac8998a73a2f312';
+
+  // @ts-ignore
+	let content :PageContent = undefined;
```
[the remark this answers is not in the document]
Comment (samijaber) [reply examples/svelte-starter-kit/src/lib/Builder.svelte:18]: oh, `PageContent` doesn't exist at all haha. I'd just set this to `any`

File: examples/svelte-starter-kit/README.md (hunk 1 above)
Comment (gustavocodess) [reply examples/svelte-starter-kit/README.md]: It was indeed outdated, just updated based on what it really is :)

File: examples/svelte-design-system/src/components/BoxWithChildren/BoxWithChildren.builder.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+export default {
+	component: BoxWithChildren,
+	name: 'BoxWithChildren',
+	canHaveChildren: true,
+	inputs: [
+		{
+			name: 'textBefore',
+			type: 'string',
+			defaultValue: 'Simple text before children'
+		},
```
Comment (teleaziz) [inline examples/svelte-design-system/src/components/BoxWithChildren/BoxWithChildren.builder.js:12]: Wdyt about declaring the `children` prop here explicity with some default value:

```
{
  name: 'children',
   type: 'blocks',
   hideFromUI: true,
   defaultValue: [
						{
							'@type': '@builder.io/sdk:Element',
							component: {
								name: 'Text',
								options: {
									text: 'this is editable within builder '
								}
							},
							responsiveStyles: {
								large: {
									display: 'flex',
									flexDirection: 'column',
									position: 'relative',
									flexShrink: '0',
									boxSizing: 'border-box',
									marginTop: '8px',
									lineHeight: 'normal',
									height: 'auto',
									textAlign: 'center'
								}
							}
						}
					]
}

```

**Review by teleaziz — APPROVED** [review]
LGTM, left few comments

File: examples/svelte-design-system/src/components/BoxWithChildren/BoxWithChildren.builder.js (hunk 3 above)
Comment (gustavocodess) [reply examples/svelte-design-system/src/components/BoxWithChildren/BoxWithChildren.builder.js:12]: Sure that looks good. I'll add that

**Pull request merged**